components/HeaderSSR/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { Container } from '@catho/quantum';
```

Comment:
The "HeaderSSR" folder seems a little redundant to me, considering that we are inside the ssr-simulator project. What do you think about just putting "Header"?